src/Inkluzitron/Modules/SystemModule.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Discord.Commands;
+using Inkluzitron.Extensions;
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace Inkluzitron.Modules
+{
+    public class SystemModule : ModuleBase
+    {
+        [Command("uptime")]
+        public async Task UptimeAsync()
+        {
+            var process = Process.GetCurrentProcess();
+            var uptime = (DateTime.Now - process.StartTime).FullTextFormat();
+            var start = process.StartTime.ToString("dd. MM. yyyy HH:mm:ss");
+            var activeTime = process.TotalProcessorTime.FullTextFormat();
+
+            await ReplyAsync($"Start proběhl **{start}**, to znamená, že běžím {uptime}, ale reálně jsem pracoval {activeTime}.");
```

Comment:
```suggestion
            await ReplyAsync($"Start proběhl **{start}** - to znamená, že běžím {uptime} (ale reálně jsem pracoval {activeTime}).");
```

Nebylo by to takhle lepší? Nejsem si jistý tou pomlčkou, ale myslím že je tam vhodnější než čárka

### Příklad výpisu 
```
Start proběhl 20.5.2021 12:51 - to znamená, že běžím 5 hodin 24 minut (ale reálně jsem pracoval 3 minuty).
```

Reply:
Se závorkami nemám problém, ale pomlčka se mi nejeví, že by zde byla na místě. Když už, tak možná jako 2 věty?

### Příklad
```
Start proběhl 20.5.2021 12:51. To znamená, že běžím 5 hodin 24 minut (ale reálně jsem pracoval 3 minuty).
```

Reply:
Jo, tečka je asi lepší volba

Reply:
```suggestion
            await ReplyAsync($"Start proběhl **{start}**. To znamená, že běžím {uptime} (ale reálně jsem pracoval {activeTime}).");
```